### PR TITLE
BOLT-10 Edits

### PR DESCRIPTION
**Pull request by shannona**
Mostly simple copy-editing and clarity, though I have one query on address types before I can close this out.

### CODE REVIEW

File: 10-dns-bootstrap.md [hunk 1]
```diff
@@ -1,61 +1,62 @@
...
 The following key-value pairs MUST be supported by a DNS seed:
 
  - `r`: realm byte, used to specify what realm the returned nodes must support (default value: 0, Bitcoin)
- - `a`: address types, used to specify what address types should be returned for `SRV` queries. This is a bitfield using the types from [BOLT 7](07-routing-gossip.md) as bit index. This condition MAY only be used for `SRV` queries. (default value: 6, i.e., `2 || 4`, IPv4 and IPv6)
- - `l`: `node_id`, the bech32 encoded `node_id` of a specific node, used to ask for a single node instead of a random selection. (default: null)
- - `n`: textual representation of the number of desired reply records (default: 25)
+ - `a`: address types, used to specify what address types should be returned for `SRV` queries. This is a bitfield that uses the types from [BOLT 7](07-routing-gossip.md) as bit index. This condition MAY only be used for `SRV` queries. (default value: 6, i.e., `2 || 4`, IPv4 and IPv6) [ FIXME: BOLT 7 says that IPv4 and IPv6 are address descriptor types 1 & 2. Are we missing something in this description? ]
```
Comment (rustyrussell) [inline 10-dns-bootstrap.md]: Ah, this is subtle.  The 2 == "bit 1" and 4 == "bit 2" (counting bits from 0).   Perhaps expand:

(default value: 6, ie. bit 1 and bit 2 set for IPv4 and IPv4 respectively)

?

File: 10-dns-bootstrap.md [hunk 2]
```diff
@@ -1,61 +1,62 @@
 # BOLT #10: DNS Bootstrap and Assisted Node Location
 
-This specification describes a node discovery mechanism based on the Domain Name System.
+This specification describes a node discovery mechanism based on the Domain Name System (DNS).
 Its purpose is twofold:
 
- - Bootstrap: the initial node discovery for nodes that have no known contacts in the network
- - Assisted Node Location: supporting nodes to discover the current network address of previously known peers
+ - Bootstrap: providing the initial node discovery for nodes that have no known contacts in the network
+ - Assisted Node Location: supporting nodes in  discovery of the current network address of previously known peers
```
Comment (toadlyBroodle) [inline 10-dns-bootstrap.md]: two spaces added

File: 10-dns-bootstrap.md [hunk 3]
```diff
@@ -1,61 +1,62 @@
...
 Since `SRV` records return a _(hostname,port)_-tuple, nodes that are listening on non-default ports MAY be returned.
 
-Upon receiving a node query, the seed MUST select the record matching the `node_id` if any, and return all addresses associated with that node.
+Upon receiving a node query, the seed MUST select the record matching the `node_id`, if any, and return all addresses associated with that node.
 
 ### Reply Construction
 
-The results are serialized in a reply with a query type matching the client's query type, i.e., `A` queries result in `A` replies, `AAAA` queries result in `AAAA` replies and `SRV` queries result in `SRV` replies, but may be augmented with additional records, e.g., to add `A` or `AAAA` records matching the returned `SRV` records.
+The results are serialized in a reply with a query type matching the client's query type, i.e., `A` queries result in `A` replies, `AAAA` queries result in `AAAA` replies, and `SRV` queries result in `SRV` replies, but they may be augmented with additional records (e.g., to add `A` or `AAAA` records matching the returned `SRV` records).
```
Comment (toadlyBroodle) [inline 10-dns-bootstrap.md]: no comma after 'e.g.'

**Comment by shannona** [comment]
Responses to comments added to pull.

File: 10-dns-bootstrap.md (hunk 1 above)
Comment (cdecker) [reply 10-dns-bootstrap.md]: Yes, that was the intention, sorry for burrying the lead :-)

**Pull request merged**